qt/scientific_interfaces/Inelastic/Common/WorkspaceManipulationUtils.cpp

```diff
@@ -0,0 +1,207 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
Should this file still exist? It looks like we have a `WorkspaceManipulationUtils.cpp` and `WorkspaceUtils.cpp` and their respective headers, but the `WorkspaceManipulationUtils` files are not in the CMakeLists

Reply:
Thanks for spotting this. It was the name I initially put to the file, but it felt too long, then I forgot to remove the file.